tensorflow/python/kernel_tests/eig_op_test.py

```diff
@@ -194,5 +262,10 @@ def Test(self):
           shape = batch_dims + (size, size)
           name = "%s_%s_%s" % (dtype.name, "_".join(map(str, shape)), compute_v)
           _AddTest(EigTest, "Eig", name, _GetEigTest(dtype, shape, compute_v))
-          # No gradient yet
+
+          # TODO: gradient_check gets wrong numeric output for real inputs
+          # (might be connected with the fact that outputs are complex)
```

Comment:
Should I open an issue about it? Should I wait before the PR is merge first then?

Reply:
This usually means that you not missing a conjugation somewhere your gradients. What exactly is the discrepancy?

Reply:
Numeric gradients for real inputs are unreasonably large (order 1e6).

Reply:
I'm pretty sure that the problem is with numeric evaluation and not with analytic one.

Reply:
Can you use an alternative implementation of numeric evaluation for this test instead of skipping the float dtypes altogether?

Reply:
Is there alternative implementation? I though to try investigate the bug in `gradient_check` after this get merged

Reply:
@alextp I've rerun tests locally and it seems to work now

Reply:
sorry its not true